habitat/tasks/rearrange/rearrange_sim.py

```diff
@@ -0,0 +1,776 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import json
+from collections import defaultdict
+from typing import Any, Callable, Dict
+
+import attr
+import magnum as mn
+import numpy as np
+
+import habitat_sim
+from habitat.core.registry import registry
+from habitat.sims.habitat_simulator.habitat_simulator import HabitatSim
+from habitat.tasks.rearrange.obj_loaders import (
+    add_obj,
+    load_articulated_objs,
+    load_objs,
+    place_viz_objs,
+)
+from habitat.tasks.rearrange.utils import (
+    convert_legacy_cfg,
+    get_aabb,
+    get_nav_mesh_settings,
+)
+from habitat_sim.gfx import LightInfo, LightPositionModel
+from habitat_sim.physics import MotionType
+from habitat_sim.robots import FetchRobot
+
+# (unique id, Filename, BB size, BB offset, Robot base offset [can be none])
+ART_BBS = [
+    ("fridge", "fridge.urdf", [0.33, 0.9, 0.33], [0, 0.2, 0], [1.033, 0.0]),
+    (
+        "counter",
+        "kitchen_counter.urdf",
+        [0.28, 0.5, 1.53],
+        [-0.067, 0.5, 0.0],
+        None,
+    ),
+    # Counter R
+    (
+        "counter_R",
+        "kitchen_counter.urdf",
+        [0.28, 0.5, 0.55],
+        [-0.067, 0.5, 1.0],
+        [0.526, 1.107],
+    ),
+    # Counter L
+    (
+        "counter_L",
+        "kitchen_counter.urdf",
+        [0.28, 0.5, 0.75],
+        [-0.067, 0.5, -0.7],
+        [0.524, -0.896],
+    ),
+]
+
+
+# temp workflow for loading lights into Habitat scene
+def load_light_setup_for_glb(json_filepath):
```

Comment:
Should be encoded in the SceneDataset for ReplicaCAD or in general the LightAttributeManager can do this for you  and register the setup in Simulator.